### ISSUE TITLE
Does not run on mac (lwjgl3)

### ISSUE THREAD

**Issue by negativeview** [tommyettinger/gdx-liftoff#2]
Mac needs -XstartOnFirstThread added to the run task. I quick hack fixed this by adding the following the lwjgl3's build.gradle. I'm sure there are cleaner solutions.

```java
import org.gradle.internal.os.OperatingSystem

run {
    switch (OperatingSystem.current()) {
        case OperatingSystem.LINUX:
            break
        case OperatingSystem.MAC_OS:
            jvmArgs '-XstartOnFirstThread'
            break
        case OperatingSystem.WINDOWS:
            break
    }
}
```

**Comment by tommyettinger** [comment]
Oh... wow, that's bad. Just so I'm clear, this is a generated project that has a bad run task, not gdx-liftoff itself? I'll look into what solutions are recommended by other libGDX/LWJGL3 projects, and I'll try to get this working tonight.

**Comment by tommyettinger** [comment]
I'm going to try attaching a pre-release version here instead of permanently adding a release for this. Here goes nothing...
[gdx-liftoff-1.9.9.2-ALPHA.zip](https://github.com/tommyettinger/gdx-liftoff/files/3269191/gdx-liftoff-1.9.9.2-ALPHA.zip)

**Comment by negativeview** [comment]
Yes. It's the generated code. I haven't done anything with liftoff from source. Will try soon. At wedding this weekend, but going home today.

**Comment by negativeview** [comment]
I'm also not sure if this affects old desktop on mac. I think it might be any gui app, not specific to version 3. I can try to confirm that for you, too, later.

**Comment by negativeview** [comment]
Told you on Discord, but to reiterate here, 1.9.9.2-ALPHA runs on mac just fine.

**Comment by negativeview** [comment]
Told you on Discord, but to reiterate here, 1.9.9.2-ALPHA runs on mac just fine.

**Comment by tommyettinger** [comment]
I forgot to close this! This is addressed in release 1.9.9.2, at least for LWJGL3, which is encouraged already anyway. The LWJGL2 (desktop) support is weirder and I don't know what is happening there.
